roles/network_plugin/calico/templates/calico-node.yml.j2

```diff
@@ -200,9 +200,12 @@ spec:
 {% endif %}
             - name: CALICO_IPV4POOL_IPIP
               value: "{{ calico_ipv4pool_ipip }}"
-            # Disable IPv6 on Kubernetes.
             - name: FELIX_IPV6SUPPORT
+{% if enable_dual_stack_networks %}
+              value: "true"
+{% else %}
               value: "false"
+{% endif %}
```

Comment:
Maybe something more like `value: "{{ enable_dual_stack_networks | default(false) }}"

Reply:
Yeah, good shout. I'll get that updated now